src/main/java/org/candlepin/subscriptions/security/SecurityConfig.java

```diff
@@ -98,7 +99,7 @@ public IdentityHeaderAuthenticationDetailsSource detailsSource(ApplicationProper
         );
     }
 
-    @Bean
+    // NOTE: intentionally *not* annotated w/ @Bean; @Bean causes an *extra* use as an application filter
```

Comment:
Would you mind putting a comment on IdentityHeaderAuthenticationFilter remarking that since that class derives from GenericFilterBean, Spring Boot registers it automatically (and drop a link to that [SO answer](https://stackoverflow.com/a/31571715))

Reply:
DONE. See new commit.